File: Android/doraemonkit/src/main/java/com/didichuxing/doraemonkit/kit/performance/PerformanceFragmentCloseListener.java
package com.didichuxing.doraemonkit.kit.performance;

/**
 * ================================================
 * 作    者：jint（金台）
 * 版    本：1.0
 * 创建日期：2019-10-11-20:39
 * 描    述：性能监控页面关闭回调接口
 * 修订历史：
 * ================================================
 */
public interface PerformanceFragmentCloseListener {

    /**
     * @param performanceTypee
     */
    void onClose(int performanceTypee);
}

File: Android/kotlin/doraemonkit/src/main/java/com/didichuxing/doraemonkit/kit/performance/manager/PerformanceFragmentCloseListener.kt
package com.didichuxing.doraemonkit.kit.performance.manager

/**
 *
 * Desc:性能监控页面关闭回调接口 kt版本
 * <p>
 * Date: 2020-06-09
 * Company:
 * Updater:
 * Update Time:
 * Update Comments:
 *
 * Author: pengyushan
 */
interface PerformanceFragmentCloseListener {
    /**
     * @param performanceTypee
     */
    fun onClose(performanceTypee: Int)
}